app/views/repos/show.json.jbuilder

```diff
@@ -0,0 +1,16 @@
+if @charts.empty?
+  json.charts []
```

Comment:
Either or would work, but `nil` makes sense to me as it doesn't exist. Empty string may imply it is not names